kim_convergence/run_length_control/core.py

```
@@ -0,0 +1,615 @@
+)
+from ._equilibration import _equilibration_stage
+from ._population import _validate_population_params
+from ._setup import _setup_algorithm
+from ._variable_list_factory import _make_variable_list
+
+__all__ = ["run_length_control"]
+
+
+def run_length_control(
+    get_trajectory: Callable,
+    get_trajectory_args: dict = {},
```

Ruff (0.14.10)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)